packages/dashboard/src/dashboard.js

```diff
@@ -43,7 +44,12 @@ import { KeyboardOnlyOutline } from './utils';
 import defaultConfig from './defaultConfig';
 
 function Dashboard({ config, children }) {
-  const _config = useMemo(() => deepMerge(defaultConfig, config), [config]);
+  // Need `cloneDeep` so that karma tests are not using a modified
+  // `defaultConfig` between runs in CI
```

Comment:
Yea so the issue is the app modifies the default config, and karma renders the app, so karma will modify the config regardless of what you do in karma land. Definitely made this issue hard to track down.

> it just seems a bit useless to clone an object in the client every time some views a page for no reason.

yeaup. agreed. we could mock defaultConfig in the fixture and just have the mock value be a deep copy of defaultConfig, that way we only experience the cost of this call in the tests and not the user facing app.

I think one thing to consider tho is that the app having the sideEffect of mutating `defaultConfig` every time feels like a bug. This would bleed over into any unit test that renders `App` with a partial config too. It's not exclusive to the karma/e2e tests

Reply:
one simple change that could fix this without a deep clone would just be to make `defaultConfig` a function vs a module scoped object.

ie
```js
const getDefaultConfig = () => ({
  // ...
  flags: { ... }
});
...
const _config = useMemo(
    () => deepMerge(getDefaultConfig(), config),
    [config]
  );
```
vs
```js
const defaultConfig = {
  // ...
  flags: { ... }
};
...
const _config = useMemo(
    () => deepMerge(defaultConfig, config),
    [config]
  );
```

Reply:
would just allocate new memory for `defaultConfig` every time the app mounts vs pointing to the same memory every time the app mounts

Reply:
Just throwing in my 🪙  🪙  I agree: 
>I think one thing to consider tho is that the app having the sideEffect of mutating defaultConfig every time feels like a bug. 

However y'all decide to go about it, I don't think that we should continue to mutate `defaultConfig`.

Reply:
Yeah we should not mutate `defaultConfig`, that was simply done by accident from what I can see.

A `getDefaultConfig()` function sounds simple enough and does not inflate bundle size.